### DISCUSSION TITLE
分享和Windows更新有關的幾點

### DISCUSSION THREAD

**Discussion by sjh501** [1101-GS4719-C/Python-Community-GS4719-C#1368] [General]
我從Windows 11發布後就使用到現在，很幸運沒有發生很大的狀況，但我有一個室友更新完後螢幕藍屏，只能重灌。想說剛好來分享一下我所知道和Windows更新有關的幾個小點：

1. Windows的更新有分幾種類型：重大更新、累積更新、安全性更新。重大更新會寫如22H1、22H2之類，22H1代表2022年的第一次、22H2則是2022年第二次，以此類推；一般來說重大更新會有比較大幅度的變化，功能的增減。累積更新就是一些優化的小更新，安全性更新就是修補漏洞之類，會用KBxxxxxxx編號。
2. 在設定 > Windows Update > 更新紀錄裡面可以看到最近的更新紀錄（好像一年內的都會有）。當更新完感覺電腦怪怪的或者直接出事，在電腦可以用的情況下我會直接將更新的代號複製到瀏覽器上面查，除了可以看到有更新了什麼，也會有災情的新聞或討論，如果跟更新很有關係就直接復原到上一個版本。
3. 如果不想常常看到更新的提醒跳出來，或者像我室友被更新搞到怕，可以到進階選項裡面把通知的選項關掉，或者乾脆點暫停更新。

如果有誤或其他沒有列出來的也麻煩大家補充~

**Comment by youtinglin** [comment]
我本人的電腦就有出現一直提醒更新，我就剛好不小心按到稍後更新，電腦隔天就變成藍屏，只好去NOVA維修重灌了，維修人員是和我說不要讓他自動更新，因為windows最近的版本相容性不太好，建議要更新直接用灌的比較好，修好電腦後也直接把電腦設定暫停更新。

**Comment by ashleyy-huang** [comment]
我覺得windows 11相對在某些功能進行了很多優化，像是記事本不用再一次開好幾個分頁，但是不知道為什麼在某些部分又相當不人性化，有些在win10可以直接找到的東西在更新後還要上網查一下他在哪裡。最後 我發現在一次新的更新之後我的影片播放器就壞掉了，上網查發現很多人也遇到這個狀況，已經過了兩個月微軟還是沒有修這個bug：（

**Comment by PaulSu0905** [comment]
我更新到WIN11之後發現有很多之前下載的軟體都還沒支援到WIN11，例如TranslucentTB是一個可以把工具列透明化的美觀用程式，就不適用於win11。害我現在不能完整地看到我美麗的IU桌布。

**Comment by Tzuhsuannnnn** [comment]
雖然win11優化了很多東西，但我覺得win 11 改動了很多設定的位置，有時候要找或改變某個設定時，都要花蠻多時間找，有點不方便。

**Comment by Brownnnn78** [comment]
自從某一次更新他把我的wifi 功能拔掉之後，我就再也沒讓他自動更新了，那時候延續了好幾個月完全沒辦法把電腦帶出門，只能插網路線，直到某次更新我的電腦才終於能夠使用wifi，從這次的更新問題才開始注意到更新雖然可能會有新的酷功能，但是電腦要能使用才最重要啊⋯⋯
